streams/src/main/java/org/apache/kafka/streams/processor/internals/StoreChangelogReader.java

```diff
@@ -81,7 +82,7 @@ public void register(final StateRestorer restorer) {
 
         final Set<TopicPartition> restoringPartitions = new HashSet<>(needsRestoring.keySet());
         try {
-            final ConsumerRecords<byte[], byte[]> allRecords = restoreConsumer.poll(10);
+            final ConsumerRecords<byte[], byte[]> allRecords = restoreConsumer.poll(Duration.ofMillis(10));
```

Comment:
10ms isn't enough time to do any real work... It seems like it should either be 0ms (async) or >= 100ms like the rest.

Reply:
I'd suggest using 100ms not 10ms, since this call is in a larger while loop in `StreamThread.runOnce`, and hence I concern doing `poll(0)` would unnecessarily increase the restoration latency.

Reply:
Shouldn't we use config `poll.ms`? Or, if present, `restore.consumer.poll.ms` ?

Reply:
Yeah, I was wondering why we sometimes use hardcoded values and other times use configured ones...